Client/Frontend/Autofill/CreditCard/CreditCardBottomSheet/CreditCardBottomSheetViewModel.swift

```diff
@@ -146,7 +163,8 @@ struct CreditCardBottomSheetViewModel {
     }
 
     // MARK: Helper Methods
-    func getPlainCreditCardValues(bottomSheetState: CreditCardBottomSheetState) -> UnencryptedCreditCardFields? {
+    func getPlainCreditCardValues(bottomSheetState: CreditCardBottomSheetState,
+                                  row: Int? = nil) -> UnencryptedCreditCardFields? {
```

Comment:
Request: Update test

Reply:
This has a test but added a ticket here for comprehensive review of this method for better testing: https://mozilla-hub.atlassian.net/browse/FXIOS-6719